13-transport.Rmd

```diff
@@ -552,7 +552,7 @@ Making the width of the routes proportional to the number of car journeys that c
 The code chunk below plots the desire lines and routes, resulting in Figure \@ref(fig:routes) which shows routes along which people drive short distances:[^13-transport-8]
```

Comment:
> The code chunk below

Something needs a change here: either drop `echo = FALSE` for the below chunk, or don't refer to the chunk that generates the plot.

Reply:
Thanks for the heads-up, will fix now.